pkg/apis/logstash/v1alpha1/logstash_types.go

```diff
@@ -42,6 +42,17 @@ type LogstashSpec struct {
 	// +kubebuilder:validation:Optional
 	ConfigRef *commonv1.ConfigSource `json:"configRef,omitempty"`
 
+	// Pipelines holds the Logstash Pipelines. At most one of [`Pipelines`, `PipelinesRef`] can be specified.
```

Comment:
Would it make sense to add a link to https://www.elastic.co/guide/en/logstash/current/multiple-pipelines.html to explain the expected format for each pipeline? (I'm a complete noob wrt Logstash, I was just trying to understand where the format was documented, so feel free to ignore this comment 😄 )

Reply:
Probably not, just like `kibana.yml` and `elasticsearch.yml` expect users to have basic context. Linking the pipeline-to-pipeline page may confuse them.